backend/src/property_handlers.py

```diff
@@ -43,12 +43,18 @@ def get_property():
                         schema: ErrorResponse
     """
     try:
-        print("sent request = " + str(request.get_json()))
         payload = GetPropertyRequest().load(request.get_json())
         _property = app.services.property_service().get_property(payload)
         _property_response = GetPropertyResponse()
         _property_response.pagination = PaginationResponse()
-        _property_response.results = PropertySchema().dump(_property)
+        _property_response.pagination = PaginationResponse(partial=True).load(
+        {
+            "results_per_page": len(_property),
```

Comment:
I know we aren't technically implementing pagination right now, but this should contain the values passed in the request